src/leaflike/bookmarks.clj

```diff
@@ -27,16 +27,33 @@
       (let [bookmark (-> (select-keys params [:title :url])
```

Comment:
`user` from https://github.com/nilenso/leaflike/blob/8f6d6627fc5ba3afbb598d806124dd29cae30d1d/src/leaflike/bookmarks.clj#L25 can go inside this `let`